scripts/setup/generate_secrets.py

```diff
@@ -33,6 +33,7 @@
     'local_database_password',
     'rabbitmq_password',
     'shared_secret',
+    'thumbor_sign_key',
```

Comment:
Given the secret names in the rest of Zulip, I think probably `thumbor_key` would be what we'd usually call such a thing; does it make sense to rename this?